plugins/zfs/zfs_list

```diff
@@ -1,21 +1,37 @@
 #!/bin/bash
-#
-# Plugin to monitor ZFS Filesystems
-# Author: Adam Michel (elfurbe@furbism.com)
-# Description:
-#  This is an extension of the zfs_fs plugin
-#  modified as a multigraph to graph all zfs
-#  filesystems it can find
-#
-# Tested on Ubuntu-14.04
-#
-# Parameters understood:
-#
-# 	config   (required)
-# 	autoconf (optional - used by munin-config)
-#
 #%# family=auto
 
+: << EOF
+=head1 NAME
+zfs_list - Plugin to monitor ZFS Filesystems
+
+This is an extension of the zfs_fs plugin
+modified as a multigraph to graph all zfs
+filesystems it can find
+
+=head1 CONFIGURATION
+plugin config:
+[zfs_list]
+env.ignore_datasets_string <Datasets to ignore; String for grep>
+
+
+=head1 AUTHOR
+=over 4
+=item * Adam Michel (elfurbe@furbism.com)
+=back
+
+
+=head1 MAGIC MARKERS
+=begin comment
+These magic markers are used by munin-node-configure when installing
+munin-node.
+=end comment
+  #%# family=auto
+=cut
+
+EOF
+
+
 . "$MUNIN_LIBDIR/plugins/plugin.sh"
 
```

Comment:
I would recommend to add something like `ignore_datasets_string=${ignore_datasets_string:-}` here. This would serve as an interface description and help users to identify the configuration variables and their default settings.

Reply:
added